### DISCUSSION TITLE
How do you enable OpenGL?

### DISCUSSION THREAD

**Discussion by linlinxza** [Legend-of-Dragoon-Modding/Severed-Chains#1110] [Q&A]
How do you enable OpenGL? I'd like to try it out. Can this be explained to me without me having to use Discord? I prefer to keep any issues I have to report on Github anyways.

**Comment by LordMonoxide** [comment]
The opengl branch has been merged back to main and is now available in the devbuild releases on github. Note there are significant rendering issues as we're still only mid-way through development, and very little work has been done on combat.

**Reply by linlinxza** [reply]
I just want to see the progress thus far for myself. I'm not expecting things to work properly yet. If I run git clone on your repo, it should have OpenGL rendering?

EDIT:

I got it compiled and working with IntelliJ. And OpenGL is working. Smoothest framerate I have ever seen for the Legend of Dragoon. Well done so far. 

Now, I won't be doing any coding (I scored a measly 60% in a Java course in college as I am not into coding; more of a hardware and networking type). But I will be reporting problems. Now, question. Is there a way to make it run with a higher resolution yet?

What I have observed so far during combat:

- Character jankiness.
- Certain effects are missing (like feathers surrounding Shana as she transforms and light effects make her look like the new dark Dragoon).

So, I see what kind of rendering issues I'll be facing. Anyway, I'll report strictly on Github for anything that is of interest for how I do my gameplay.
